docs/docs/contributing/local-backend.mdx

```diff
@@ -26,6 +26,8 @@ pip install -r requirements/testing.txt
 pip install -e .
 
 # Initialize the database
+# Note: For generating a SECRET_KEY if you haven't done already, you can use the command:
+# echo "SECRET_KEY='$(openssl rand -base64 42)'" | tee -a superset_config.py
```

Comment:
If running with `FLASK_ENV = development` this shouldn't be necessary for running

Reply:
> If running with `FLASK_ENV = development` this shouldn't be necessary for running, but still good to have this so users aren't confused when trying to run prod

@nytai Yeah. that was the idea in the change that @dpgaspar made in #23186, but it's not working, as per my issue #23288

Maybe you have an idea how to make #23186 work as intended.